src/vs/workbench/contrib/terminal/browser/terminal.contribution.ts

```diff
@@ -552,16 +552,18 @@ actionRegistry.registerWorkbenchAction(new SyncActionDescriptor(FindPrevious, Fi
 	mac: { primary: KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.KEY_G, secondary: [KeyMod.Shift | KeyCode.F3, KeyCode.Enter] },
 }, KEYBINDING_CONTEXT_TERMINAL_FIND_WIDGET_FOCUSED), 'Terminal: Find previous', category);
 
-// Commands miht be affected by Web restrictons
+// Commands might be affected by Web restrictons
 if (BrowserFeatures.clipboard.writeText) {
 	actionRegistry.registerWorkbenchAction(new SyncActionDescriptor(CopyTerminalSelectionAction, CopyTerminalSelectionAction.ID, CopyTerminalSelectionAction.LABEL, {
 		primary: KeyMod.CtrlCmd | KeyCode.KEY_C,
+		secondary: [KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.KEY_C],
```

Comment:
This will bind cmd+shift+c on mac, I think we need to define just for windows and include a primary and secondary (like linux has its own definition).

Reply:
I believe I have fixed that issue.